libs/host/Configuration/Options.cs

```diff
@@ -233,6 +233,9 @@ internal sealed class Options
         [IntRangeValidation(0, int.MaxValue)]
         [Option("compaction-freq", Required = false, HelpText = "Background hybrid log compaction frequency in seconds. 0 = disabled (compaction performed before checkpointing instead)")]
         public int CompactionFrequencySecs { get; set; }
+        [IntRangeValidation(0, int.MaxValue)]
+        [Option("hcollect-freq", Required = false, HelpText = "Frequency in seconds for the background task to perform Hash collection. 0 = disabled. Hash collect is used to delete expired fields from hash without waiting for a write operation.")]
```

Comment:
Add to comment: "Use the HCOLLECT API to collect on-demand"

Reply:
Done